src/Facebook/InstantArticles/Elements/Audio.php

```diff
@@ -212,4 +217,14 @@ public function toDOMElement($document = null)
 
         return $element;
     }
+
+    /**
+     * Overrides the @see Element::isValid().
+     *
+     * @return true for valid Audio that contains not empty url, false otherwise.
+     */
+    public function isValid()
+    {
+        return !Type::isTextEmpty($this->url);
```

Comment:
We should check if the URL is valid also. If you think that it is out of the scope for this PR, please add a TODO comment.